lib/takeaway.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'send_sms'
+
+class TakeAway
+
+  def initialize
+    @menu = { "chips" => 2.5, "kebab" => 5.5, "burger" => 4.75 }
+    @basket = []
+  end
+
+  def display_menu
+    puts @menu
+  end
+
+  def select(food_item, qty = 1)
+    @basket << { food_item => qty }
+    "You have added #{food_item} x#{qty} portions"
+  end
+
+  def summary
```

Comment:
Does the user ever get to see the total they're spending?

Reply:
I have just updated (and pushed) my summary method to show total spent so far. As a result I changed the @basket to a hash, rather than the array it was before. Feels better!